presto-spi/src/main/java/com/facebook/presto/spi/type/SqlTimestampWithTimeZone.java

```diff
@@ -26,7 +26,9 @@
 
 public final class SqlTimestampWithTimeZone
 {
-    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("uuuu-MM-dd HH:mm:ss.SSS VV");
+    // This needs to be Locale-independent, work the same in Joda and Java Time and should never change, as it defines the external API data format.
```

Comment:
seems unrelated. Extract.

Reply:
This is related. I needed this pattern in `TestingPrestoClient` to correctly parse TS W/TZ values returned from Presto API

Reply:
kk